wca-guidelines.md

```diff
@@ -205,10 +205,16 @@ To be more informative, each Guideline is classified using one of the following
-- E2c++) [CLARIFICATION] The competitor's solution must only use moves that are exactly defined in [Regulation 12a](regulations:regulation:12a). Examples of notation and moves that are not permitted in competition: [F], [R, U], [R: U], R'2, L'w, f, M, U2'.
-- E2c+++) [ADDITION] If the competitor forgot to write their name on their intended solution paper before the time limited, they are permitted to add their name in front of a judge. In this case, the competitor must ask the judge while submitting their solution and then immediately write their name (and make no other changes) under direct supervision of the judge. Once a competitor has finished submitting their solution (e.g. placing it in the judge's hands and allowing the judge to move on to the next competitor, placing it on the judge's table and walking away), they are not permitted to add their name. If the competitor writes their name after the time limit but before asking the judge, the attempt remains disqualified (DNF).
+- E2c+) [CLARIFICATION] Although the judge may provide a standard sheet for competitors to submit solutions, a competitor may submit a solution on a different piece of paper. (Note that the piece of paper must come from the judge, according to [Regulation E3a](regulations:regulation:E3a).)
+- E2c1+) [ADDITION] If the competitor forgot to write information to identify the attempt on their intended solution paper before the time limit, they are permitted to add it in front of a judge. In this case, the competitor must ask the judge while submitting their solution and then immediately write information to identify the attempt (and make no other changes) under direct supervision of the judge. Once a competitor has finished submitting their solution (e.g. placing it in the judge's hands and allowing the judge to move on to the next competitor, placing it on the judge's table and walking away), they are not permitted to add any information. If the competitor writes anything after the time limit but before asking the judge, the attempt remains disqualified (DNF).
+- E2c1++) [EXPLANATION] The competitor is not required to submit a paper with the competition name, round, and/or attempt number, but may wish to do so in order to help the organization team keep track of submitted solutions.
+- E2c1+++) [ADDITION] If a judge finds a submitted solution without information to identify the attempt (e.g. while grading the attempt), they should find out the competitor who wrote the solution in order to assign them a DNF (rather than DNS). Note that the competitor must not be allowed to write information to identify the attempt at this point, so they must receive a DNF result.
```

Comment:
E2c1+++ - I think if they are able to find out who did the attempt, it shouldn't be DNFed so I'm not for this change

Reply:
I think we need to be explicit about what is allowed to avoid inconsistent rulings from Delegates. The incidents from this year suggest that not every Delegate is on the same page about this right now.

@AlbertoPdRF, would you be in favor of allowing the result to count at the discretion of the Delegate?

Reply:
If you can find out who did the attempt, you know who did the attempt, you have their attempt - it shouldn't be a DNF. That's just being way too strict and I think it's unnecessary to be that strict in FMC

Reply:
Hmm, I just double-check and we had a unanimous WRC consensus for "Can be added in front of judge while submitting" (as opposed to the explicit alternative "Can be added until the end of the competition"): https://github.com/thewca/wca-regulations/issues/655#issuecomment-443497338

Would you be alright sticking with that, or is it possible some of us didn't take into account this case while voting?

Reply:
I'd want it clarified that if the delegate can determine whose attempt it was to put the DNF in, why not put in the time? I guess when voting I completely overlooked that option. It just doesn't make any sense to me that if fit can be determined who did the attempt, that it gets entered as a DNF. If someone leaves and they can't confirm with the competitor that it is indeed their attempt, then sure, DNF the attempt, but if you can confirm it was there, I think it should be added.

Reply:
Allowing this at the discretion of the WCA Delagate shouldn't be an option in my opinion, as we would still have the same inconsistent rulings.